src/components/combo/themes/shared/combo.bootstrap.scss

```diff
@@ -128,4 +141,25 @@ $dropdown-theme: dropdown-theme.$bootstrap;
             color: inherit;
         }
     }
+
+    ::slotted([slot='suffix']),
+    [part|='toggle'] {
+        border-inline-end: .0625rem solid var(--disabled-border-color);
+    }
+
+    ::slotted([slot='prefix']) {
+        border-inline-start: .0625rem solid var(--disabled-border-color);
+    }
 }
+
+:host(:not([disabled])),
+:host(:not(:disabled)) {
+    ::slotted([slot='suffix']),
+    [part|='toggle'] {
+        border-inline-end: .0625rem solid var(--border-color);
+    }
+
+    ::slotted([slot='prefix']) {
+        border-inline-start: .0625rem solid var(--border-color);
+    }
```

Comment:
I guess you're trying to fix a regression with this code here? I.e. it's not related to the actual bug this PR addresses?